### PR TITLE
Code coverage

### PR DESCRIPTION
**Pull request by laedit**
Trying to add code coverage in the Team City process.

### CODE REVIEW

File: build.proj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="4.0" DefaultTargets="Test;Chocolatey" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0" DefaultTargets="Compile;Test;Chocolatey" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (shiftkey) [inline build.proj]: Is there a particular reason why you're explicitly calling `Compile` here instead of the `DependsOnTargets` attribute below

Comment (laedit) [inline build.proj]: I am trying to separate the compile and test in different build step on Team City in order to activate the code coverage.

Comment (shiftkey) [inline build.proj]: :cool:

**Comment by laedit** [comment]
It works!
Now we have code coverage: http://teamcity.ginnivan.net/viewLog.html?buildId=9732&tab=coverage_dotnet&buildTypeId=bt52
If you are ok with it, I'll merge the branch.

**Comment by laedit** [comment]
Minor loss: the results for tests with theory attribute and multiple InlineData are not displayed for each InlineData detail but with the number of run:
before:
OK 	SiteContextGeneratorTests. site_context_pages_have_date_in_bag(fileName: "C:\TestSite\2014-01-01-ByFilename.md", useDefault: False) (Pretzel.Tests.dll:Pretzel.Tests.Templating.Context)
OK 	SiteContextGeneratorTests. site_context_pages_have_date_in_bag(fileName: "C:\TestSite\UsingDefault.md", useDefault: True) (Pretzel.Tests.dll:Pretzel.Tests.Templating.Context) 

After:
OK (2 runs) 	SiteContextGeneratorTests. site_context_pages_have_date_in_bag (Pretzel.Tests.dll:Pretzel.Tests.Templating.Context)

**Comment by JakeGinnivan** [comment]
Looks good to me, merge away :)

**Comment by laedit** [comment]
Merge done.
Tests from all previous PRs will fail, but it can be "fixed" if necessary by disable the "Compile" and "Test" build steps and enable "Version Build & Test".

**Pull request merged**